sift/version.py

```diff
@@ -1,3 +1,3 @@
 # NB: Be sure to keep in sync w/ setup.py
-VERSION = '1.1.2.1'
+VERSION = '1.1.2.2'
```

Comment:
This should match setup.py

Is there a convention for what updating the 1st vs. 4th digit means in terms of backwards compatibility